ERROR    root:init_helpers.py:90 Failed to capture screenshot: [Errno 2] No such file or directory: 'data/attachments/dev_chrome_130.0.6723.58_user_should_be_logged_out.png'